ui/main/package.json

```diff
@@ -45,6 +45,7 @@
     "bootstrap": "5.2.0",
     "chart.js": "3.9.1",
     "chartjs-adapter-date-fns": "2.0.0",
+    "chartjs-plugin-datalabels": "^2.1.0",
```

Comment:
use the precise version of the component (this is to avoid delivering in a release a version that has not been fully tested) 
So :

`"chartjs-plugin-datalabels": "2.1.0"`

Reply:
Thanks, I changed it!